dspace-api/src/main/java/org/dspace/core/AbstractHibernateDAO.java

```diff
@@ -105,127 +106,87 @@ public T findByID(Context context, Class clazz, int id) throws SQLException {
      */
     public List<T> findMany(Context context, Query query) throws SQLException {
         @SuppressWarnings("unchecked")
-        List<T> result = (List<T>) query.list();
+        List<T> result = (List<T>) query.getResultList();
         return result;
     }
 
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass);
-    }
-
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @param alias           alias for the type.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass, alias);
-    }
+//    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass);
+//    }
+//
+//    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass, alias);
+//    }
 
-    /**
-     * Create a parsed query from a query expression.
-     *
-     * @param context current DSpace context.
-     * @param query   textual form of the query.
-     * @return parsed form of the query.
-     * @throws SQLException
-     */
```

Comment:
Is there a reason why we are stripping out method comments in these updated methods?